examples/NuPIC Walkthrough.ipynb

```diff
@@ -674,7 +674,7 @@
    "outputs": [],
    "source": [
     "# Step 1: create Temporal Pooler instance with appropriate parameters\n",
-    "tp = TP(numberOfCols=50, cellsPerColumn=2,\n",
+    "tm = TP(numberOfCols=50, cellsPerColumn=2,\n",
```

Comment:
I think so (with `BacktrackignTM`). I fixed the notebook, and everything executes properly now.